config/jobs/kubernetes/sig-aws/sig-aws-credentials.yaml

```diff
@@ -0,0 +1,30 @@
+presets:
+# Credentials for using AWS test account 768319786644. Used for kops CNCF tests.
+- labels:
+    preset-aws-credential: "true"
```

Comment:
do you feel like to rename the labels? aka, aws-cncf, and aws-eks?

Reply:
Yeah.. can do that in a follow-up PR (as many jobs will need to be updated).

Reply:
As promised - https://github.com/kubernetes/test-infra/pull/10893